classes/Tools.php

```diff
@@ -4307,6 +4309,40 @@ public static function isFileFresh(string $file, int $timeout = self::CACHE_LIFE
 
         return false;
     }
+
+    /**
+     * @return bool
+     */
+    public static function isDetectCountryFromBrowserLanguageAvailable(): bool
```

Comment:
```suggestion
    public static function detectCountryFromBrowserLanguageWhenAvailable(): bool
```
This function detects the country using the browser language when available
⬇️ 
detectCountryFromBrowserLanguageWhenAvailable

📘 😊

Reply:
This method just confirm that whole informations are available to deal with. So it returns a boolean. I'm ope to rename it, but removing the 'is' hide the fact that just return a boolean.

Reply:
I agree with this ! I will apply it.